app/Helpers/CheckRateableHelper.php

```diff
@@ -2,8 +2,8 @@
 
 namespace App\Helpers;
 
-class CheckRateable{
-    public static function checkRateable($content_type, $content_id, $user_id){
+class CheckRateable {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition

<!-- Comment by [SideCI](https://sideci.com) -->